app/(main)/league/[leagueId]/entry/all/page.test.tsx

```diff
@@ -24,6 +24,26 @@ jest.mock('@/api/apiFunctions', () => ({
   ),
 }));
 
+jest.mock('@/components/LeagueEntries/LeagueEntries', () => ({
+  LeagueEntries: ({
+    isPickSet,
+    teamLogo,
+  }: {
+    isPickSet: boolean;
+    teamLogo?: string;
+  }) => (
+    <div data-testid="league-entry">
+      {!isPickSet && <span data-testid="make-pick-text">Make Pick</span>}
```

Comment:
nit: If we follow BEM naming convention, this testid should be `league-entry__make-pick-text`

Reply:
Thank you for the guidance and updated!